pkg/build-rpm.sh

```
@@ -0,0 +1,42 @@
+#!/bin/bash
+set -euo pipefail
+
+RHEL="$(rpm --eval %rhel)"
+
+prepare() {
+  setup_dnf_build_env
+  echo "Copying packaging files..."
+  cp ${COMPONENT_NAME}/rpm/snowflake.spec ~/rpmbuild/SPECS/
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)